core/quantity/src/Formatter/Interfaces.ts

```diff
@@ -25,9 +25,13 @@ export interface FormatProps {
   readonly scientificType?: string; // conditionally required
   readonly stationOffsetSize?: number; // conditionally required
   readonly stationSeparator?: string;
+  readonly azimuthBase?: number; // value is specified from east counter-clockwise
```

Comment:
should prob add comments above using `/** ... */` syntax so they show up in vscode, this can be done in a separate pr.

Reply:
changed it.